boilerplate/Jenkinsfile

```diff
@@ -13,7 +13,7 @@ pipeline {
     parameters {

```

Comment:
we don't use Jenkinsfiles anymore. Let's just get rid of it

Reply:
done in develop